UI-session-7/javascript.js

```diff
@@ -0,0 +1,356 @@
+var employeeName, employeeGender, employeeEmail, 
+employeePassword, 
+employeeMatchPassword, employeeNumber,currentField,
```

Comment:
Avoid using global variables

Reply:
minimized the global variable , only "currentField" and "currentForm" are set as global since they are used in the entire file repeatedly